mobile/src/calendar/calendar-screen.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { Component } from 'react';
 import { View, Text } from 'react-native';
 import styles from '../layout/styles';
+import { Days } from './days/days';
 
 export class CalendarScreen extends Component {
```

Comment:
what do you think about renaming the whole folder to `Schedule` or `EmployeeCalendar`? I'm confused how you're going to name the calendar part of the calendar screen :) But it's probably better to leave it as is, since the screen component is named calendar-screen, not just calendar...